src/Pages/Sender/EnterPrice.tsx

```diff
@@ -60,6 +60,8 @@ const EnterPrice: React.FC = () => {
               value={objecInfo.price !== 0 ? objecInfo.price : undefined}
               onChange={handleInputChange}
               placeholder={t('requestCost.pricePlaceholder') || ''}
+              suffix="CHF"
```

Comment:
The suffix is cool to have! We could even get rid of the "(in CHF)" in the title...